src/main/scala/is/hail/utils/PrintConfig.scala

```diff
@@ -0,0 +1,8 @@
+package is.hail.utils
+
+object PrintConfig {
+  val default: PrintConfig = PrintConfig(missing="NA", boolTrue = "true", boolFalse = "false",
+    floatFormat = "%.4e")
```

Comment:
My hairs stand up at four digits of precision. This is barely enough to unambiguously represent a 16-bit float (consider the size of the mantissa). 8 digits is sufficient for a 32-bit float. 16 digits is enough for a 64-bit float.

Reply:
this is 5 digits -- 4 after the decimal point. I'd argue the contrary -- even though this can't unambiguously represent the floating point numbers it's generated from, the data it's meant to represent is adequately represented because _hardly ever_ do we have a value where the relative error is less than 0.001%.

Reply:
I am still reticent, but let's stick to your intuition about our users' use cases.